pytorch_lightning/utilities/cli.py

```diff
@@ -889,9 +893,11 @@ def instantiate_class(args: Union[Any, Tuple[Any, ...]], init: Dict[str, Any]) -
 
 
 def _get_short_description(component: object) -> Optional[str]:
-    parse, _ = import_docstring_parse("LightningCLI(run=True)")
-    try:
-        docstring = parse(component.__doc__)
-        return docstring.short_description
-    except ValueError:
-        rank_zero_warn(f"Failed parsing docstring for {component}")
+    if not docstring_parser:
+        rank_zero_warn(f"Failed parsing docstring for {component}: docstring-parser package is required")
+    else:
+        try:
+            docstring = docstring_parser.parse(component.__doc__)
```

Comment:
We should resolve failing mypy check, which is saying, that `docstring_parser.parse` expects `str`, but we are passing `Optional[str]`

Reply:
mypy still fails, but now due to the optional import. Will fix it.